config/default.json

```diff
@@ -187,7 +187,7 @@
   ],
   "pact": {
     "branchName": "Dev",
```

Comment:
This branch name needs to be Master and in the pact publish you need to check logic publish only if its master

Reply:
updated branch name to master

Reply:
added logic in publish.ts to publish only if branchName/pactTag returned is "master"